packages/cli/src/generated/object-set-watcher-api/object/FoundryObject.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { DatasourceRid } from "./DatasourceRid.js";
+import type { ObjectKey } from "./ObjectKey.js";
+import type { ObjectTypeRid } from "./ObjectTypeRid.js";
+import type { ObjectVersion } from "./ObjectVersion.js";
+
+export interface FoundryObject {
+  type: ObjectTypeRid;
+  key: ObjectKey;
+  datasources: Array<DatasourceRid>;
+  properties: /* OOOOOOPS */ any;
```

Comment:
Are we worried about these being unhandled?

Reply:
Just saw this as well. Updating my generator.